features/shipping/managing_shipments/shipping_a_shipment_from_shipment_list.feature

```diff
@@ -23,14 +23,14 @@ Feature: Shipping a shipment from shipment list
         And an email with the "UPS" shipment's confirmation for the "#00000001" order should be sent to "donald@duck.com"
         And I should see the shipment of order "#00000001" as "Shipped"
 
-    @ui @email
+    @ui @api @email
     Scenario: Shipping a shipment with tracking code from shipments index
         When I browse shipments
         And I ship the shipment of order "#00000001" with "AWDDXS-SAAQQ-SEFFX-CCDSE" tracking code
         Then I should be notified that the shipment has been successfully shipped
         And an email with the shipment's confirmation of the order "#00000001" should be sent to "donald@duck.com"
 
-    @ui @api @email
```

Comment:
Why have you deleted this tag?

Reply:
in this PR I fixed comment from #11230, here I added by mistake `@email` tag so now i have removed it (this scenario does not have any email-related steps)